internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -96,14 +98,28 @@ type EnvConfig struct {
 	ForceUpdate         bool
 }
 
+func (cfg *EnvConfig) loadCustomResourceURLs(crs []uploadable) error {
```

Comment:
what do you think of this signature?
```go
func (cfg *EnvConfig) loadCustomResourceURLs(crs []uploadable) (map[string]string, error) 
```

This way, in `NewEnvStackConfig(input *EnvConfig) (*Env, error)`, we can do
```go
urls, err := input.loadCustomResourceURLs(uploadableCRs(crs).convert());
return &Env{
		in:                in,
		prevParams:        prevParams,
		lastForceUpdateID: lastForceUpdateID,
		parser:            fs,
                customResourceURLs: urls
	}, nil
```

It's slightly more obvious what fields are set and what fields are not this way

Reply:
We discussed offline! the `load` method's receive is the runtimeConfig or the inputConfig for an environment and not `Env` itself